test/distributed/_shard/sharded_tensor/ops/test_math_ops.py

```diff
@@ -16,19 +15,17 @@
 )
 
 from torch.testing._internal.distributed._shard.sharded_tensor import (
-    TEST_GPU_NUM,
     ShardedTensorTestBase,
     with_comms,
 )
 
 from torch.testing._internal.distributed._shard.sharded_tensor._test_ops_common import (
-    gen_binary_op_func,
-    generate_chunk_sharding_specs_for_test,
+    gen_binary_op_func
 )
 
 class TestMathOps(ShardedTensorTestBase):
     @with_comms(init_rpc=False)
-    @skip_if_lt_x_gpu(TEST_GPU_NUM)
+    @skip_if_lt_x_gpu(4)
```

Comment:
Why change this to 4? Isn't `TEST_GPU_NUM` more generic?

Reply:
Opps, I think I use `TEST_GPU_NUM ` originally. Let me change it.